### ISSUE TITLE
untuple fails

### ISSUE THREAD

**Issue by amieres** [dotnet-websharper/websharper#896]
```
let a, b = 1, 2
```
Fails with the following error:

```
WebSharper error at line 24:Method name not found in JavaScript compilation: (patternInput@24-8 : unit -> (System.Int32 * System.Int32)), Members: get_Tasks, get_NewTaskName, get_b, get_a, get_Main, Main$41$28, Main$38$17, Main$32$31, Main, a, b, NewTaskName, Tasks
WebSharper error at line 24:Method name not found in JavaScript compilation: (patternInput@24-8 : unit -> (System.Int32 * System.Int32)), Members: get_Tasks, get_NewTaskName, get_b, get_a, get_Main, Main$41$28, Main$38$17, Main$32$31, Main, a, b, NewTaskName, Tasks
```

Tested in try.websharper.com:
http://try.websharper.com/snippet/user3359/0000Ia

**Comment by amieres** [comment]
I see it is already reported

**Comment by Jand42** [comment]
@amieres Thanks for the report! I have created a new ticket #897 about it, because this was indeed a limitation in WebSharper 3 because of the F# compiler itself, but is an oversight in WebSharper 4.

**Comment by Jand42** [comment]
@amieres Fix is now released in 4.1.6.207.

**Comment by amieres** [comment]
Thank you!
